client/signup/config/flows.js

```diff
@@ -69,6 +69,7 @@ const flows = {
 		lastModified: '2016-06-02',
 		meta: {
 			skipBundlingPlan: true,
+			enableStore: true,
```

Comment:
Let's not enable store for `/start/business` right now. It doesn't work 100%. The plans step is still shown. And it currently must be because we are assigning `is_store_signup` to the cart which is being sent to our servers during the `plans-atomic-store` step. We need it to know that we should do AT on the background. See Slack for more info.